app/query/query.go

```diff
@@ -1,17 +1,18 @@
-package proxy
+package query
```

Comment:
Not sure how I feel about the package being called query, which is just 

> a wrapper around client JSON-RPC query

The meat of the package logic seems to be in `Caller` but naming the package after it doesn't sound right either. Maybe proxy? 😂